python_src/evolution/random_evolution.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from deap import tools, creator, base
+from threading import Thread, Event
+
+# Event object used to send signals from one thread to another
+STOP_EVENT = Event()
+FIRST = 0
+
+
+class RandomEvolution:
+    def __init__(self, orig_ind, fitness, generate_random_ind, select, mutate, expectations):
+        creator.create("FitnessMax", base.Fitness, weights=(1.0,))
+        creator.create("Individual", list, fitness=creator.FitnessMax)
+
+        self.toolbox = base.Toolbox()
+        # Attribute generator
+        self.toolbox.register("random_ind", generate_random_ind, orig_ind, params={"min": 10, "max": 50, "dim": 1})
```

Comment:
Are those the parameters that define min and max speed? Is 50 a sensible limit? What's `dim`?

Reply:
Those parameters are refactored into variables. Now we can define those parameters while creating Random Evolution object.